packages/server/api/webscrape.py

```diff
@@ -1,20 +1,40 @@
 from http.server import BaseHTTPRequestHandler
+import urllib.request as request
+import re
 from urllib.parse import parse_qs
+from bs4 import BeautifulSoup
 
 class handler(BaseHTTPRequestHandler):
-
+  whitelisted_url_regexes = [
+    re.compile("^https:\/\/www.bonappetit.com")
+  ]
   def do_GET(self):
     queryString = self.path.split('webscrape?')
     if(len(queryString) > 1 and 'url=' in queryString[1]):
-        query = parse_qs(queryString[1])
-        self.send_response(200)
+      query = parse_qs(queryString[1])
```

Comment:
Also, 1 is a magic number here. I'd prefer having it in a constant variable with a descriptive name